cfbs/commands.py

```diff
@@ -65,6 +65,7 @@
 from cfbs.git_magic import Result, commit_after_command, git_commit_maybe_prompt
 from cfbs.prompts import YES_NO_CHOICES, prompt_user
 from cfbs.module import Module, is_module_added_manually
+from masterfiles.generate_release_information import generate_release_information
```

Comment:
When masterfiles is inside cfbs, it'll look like this:

```suggestion
from cfbs.masterfiles.generate_release_information import generate_release_information
```

Reply:
Done in https://github.com/cfengine/cfbs/pull/208/commits/3f0f1ed42b9971a87d0bf816fe3e9ed416fd7371